common/segment_trait_error.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+parent: Common
+---
+
+# Segment Trait Error
+
+Understanding the `SegmentTraitError` exception.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .warning }
+> _A **Segment-Trait Combination** unknown to pyracf would generate an improper request to IRRSMO00. Pyracf will always raise a `SegmentTraitError` to bring attention to any unknown values prior to executing such a request._
```

Comment:
I don't think a warning annotation is necessary here.

Reply:
Fine by me, my intention was consistency with the existing error types. This warning has been removed.